### PR TITLE
Add miniFasta to conda-forge

### PR DESCRIPTION
**Pull request by not-a-feature**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by not-a-feature** [comment]
@conda-forge/help-python, ready for review!

File: recipes/miniFASTA/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+
+test:
+  imports:
+    - miniFasta
```
Comment (BastianZim) [inline recipes/miniFASTA/meta.yaml:26]:
```suggestion
test:
  imports:
    - miniFasta
  commands:
    - pip check
  requires:
    - pip
```
This just double checks the reqs.

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just one small thing then this is ready to go.

File: recipes/miniFASTA/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+about:
+  home: https://github.com/not-a-feature/miniFASTA
+  license: GPL-3.0
```
Comment (BastianZim) [inline recipes/miniFASTA/meta.yaml:30]: As the linter said, this license is not acceptable. Can you please clarify whether you are using `GPL-3.0-only` or `GPL-3.0-or-later`?

Ref: https://spdx.org/licenses/

Comment (not-a-feature) [reply recipes/miniFASTA/meta.yaml]: Good point. I changed that.

**Pull request merged**